experiment/kind-conformance-e2e.sh

```diff
@@ -21,10 +21,6 @@ set -o xtrace
 source $(dirname "${BASH_SOURCE[0]}")/kind-e2e.sh
 
 run_tests() {
-  # binaries needed by the conformance image
-  rm -rf _output/bin
-  make WHAT="test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo cmd/kubectl"
-
```

Comment:
This section seemed unnecessary.

This script is sourcing `kind-e2e.sh` and running `main` which runs the `build` command which uses bazel to build all these things. This was just _re-building_ them as far as I can tell.